style.css

```diff
@@ -360,3 +389,178 @@ body {
   margin-bottom: 0.8px;
   margin-top: 2rem;
 }
+
+/* desktop version css styles */
+
+@media screen and (min-width: 768px) {
+  .navbar {
+    display: none;
+  }
+
+  .links {
+    display: flex;
+    gap: 3.2rem;
+    font-family: "Poppins", sans-serif;
+    font-weight: 500;
+    font-size: 1.5rem;
+    line-height: 2rem;
+    color: #344563;
+  }
+
+  .links li {
+    list-style: none;
+  }
+
+  .links a {
+    color: #344563;
+    text-decoration: none;
+  }
+
+  .headilne-desk {
+    background-image: url("assets/images/Header bg-desk.svg");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    height: 721px;
+    padding-top: 11.8rem;
+    padding-left: 3.46rem;
+    padding-right: 0.8rem;
+  }
+
+  .introduction {
+    width: 60%;
+    margin: 0 auto;
+  }
+
+  .heading {
+    display: none;
+  }
+
+  .heading-desk {
+    display: block;
+    font-size: 4.8rem;
+    line-height: 6rem;
+    letter-spacing: 0.037rem;
+    margin-bottom: 0;
+  }
+
+  .intro-info {
+    font-size: 2rem;
+    line-height: 2.8rem;
+  }
+
+  .projects {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 80%;
+    margin: 0 auto;
+  }
+
+  .project-1-desktop {
+    display: inline-block;
+  }
+
+  .projects-two-desk img {
+    order: 1;
+  }
+
+  .project {
+    display: none;
+  }
+
+  .title {
+    line-height: 5.2rem;
+    font-size: 4rem;
+  }
+
+  .title-mobile {
+    display: none;
+  }
+
+  .title-desk {
+    display: block;
+  }
+
+  .categories {
+    font-size: 1.8rem;
+    line-height: 2.4rem;
+  }
+
+  .categories-mobile {
+    display: none;
+  }
+
+  .categories-desk {
+    display: flex;
+  }
+
+  .intro-works {
+    line-height: 2.4rem;
+    font-size: 1.6rem;
+  }
+
+  .languages {
+    font-size: 1.2rem;
+    line-height: 1.6rem;
+    margin-bottom: 4rem;
+  }
+
+  .lang-desk {
+    display: inline;
+  }
+
+  /* About me section */
+  .about-me {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    padding-top: 14.2rem;
+  }
```

Comment:
- [ ] Kindly make sure to provide enough padding on both sides of the `About-me` section in order to match the Figma design as much as possible. 

- [ ] Kindly make sure your `Get my resue` button width is the same as the one Figma design.

  - ### YOUR DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/220702634-64ed78d2-50b5-49ce-b8da-d3224cfcc281.png)

  - ### FIGMA DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/220702855-f77eb26d-8c28-4270-a511-61ab149ad671.png)